tests/ACPPlacesMonitorTests.m

```diff
@@ -73,7 +73,7 @@ - (void) testRegisterExtensionFailure {
     // verify
     OCMVerify([_coreMock log:ACPMobileLogLevelError
                          tag:ACPPlacesMonitorExtensionName_Test
-                     message:@"An error occurred while attempting to register the ACPPlacesMonitor extension: unknown error"]);
+                     message:@"An error occurred while attempting to register the ACPPlacesMonitor extension: unknown error. For more details refer to https://docs.adobe.com/content/help/en/places/using/places-ext-aep-sdks/places-monitor-extension/places-monitor-api-reference.html#registerextension-ios"]);
```

Comment:
Replace link with ACPPlacesMonitorRegisterExtensionDocs?

Reply:
This is the unit test, not the actual code.